### ISSUE TITLE
RHEL repository for libzbxpgsql

### ISSUE THREAD

**Issue by dfisek** [cavaliercoder/libzbxpgsql#33]
There are RHEL packages for libzbxpgsql in Sourceforge.

It would be better if these packages were served as a package repository, so the repository could be added and updates in the future could be managed by yum automatically.

**Comment by cavaliercoder** [comment]
Agreed. I just haven't found any file host which would provide nice download statistics and a badge/hit counter for the Github page.

Any recommendations?

**Comment by dfisek** [comment]
How about starting without any Github download statistics/counter, just with a simple http or ftp host? :)

OpenSuSE has a build service (OBS - https://build.opensuse.org/). It builds packages not just for SuSE but for many distributions. It also provides a public repository for the packages you build.

My suggestion would be to try out OBS.

**Comment by jangaraj** [comment]
@cavaliercoder What about packagecloud.io? I would like to start my Zabbix community package repo:
![capture](https://cloud.githubusercontent.com/assets/1182932/13549520/f129b2f2-e306-11e5-93c6-91dba7972b2b.PNG)

Packagecloud is a service designed for world wide distribution of rpm/deb/... packages. Statistics per repo/package, CI integration (Travis included), user management are there. Unfortunately  I don't badge/hit counter for the Github page (we can ask support or ask for this feature).

Idea is to have Zabbix community package repo, so Zabbix user will install only community repo once. Then it'll be only  `apt-get/yum install -y <package>`. All packages must meet some minimal quality level (for example CI/CD process).

Ryan, it will be an honour to publish your libzbxpgsql as a first package. Then I would like to publish my Zabbix docker module there as well. If it sounds good, please email me.

**Comment by cavaliercoder** [comment]
@jangaraj that sounds perfect. Count me in for your community repo. Thanks very much.

**Comment by volga629** [comment]
Hello Everyone,
Is possible share srpm that  will be possible generate rpm for Fedora Server

**Comment by cavaliercoder** [comment]
@volga629 we'll share SRPMS also if we create a repo. In the meantime, the RPM spec can be found here: https://github.com/cavaliercoder/libzbxpgsql-build/tree/master/rpmbuild

**Comment by volga629** [comment]
Thank you, I  will post  rpm  and srpm at http://ftpsrv01.networklab.ca/fedora/25/RPMS/x86_64/
